sda-commons-dependencies/build.gradle

```diff
@@ -48,6 +48,9 @@ dependencies {
     api "net.javacrumbs.json-unit:json-unit-core:$jsonUnitVersion", {
       because "conflict between com.github.tomakehurst:wiremock-jre8 and net.javacrumbs.json-unit:json-unit-assertj"
     }
+    api "io.dropwizard.logback:logback-throttling-appender:1.1.3", {
```

Comment:
Nice! Do we want to add comment so we don't forget to remove it after the next Dropwizard update?

Reply:
Sounds good, I added a comment accordingly.